man/signal-cli-dbus.5.adoc

```diff
@@ -29,126 +29,252 @@ method(arg1<type>, arg2<type>, ...) -> return<type>
 
 Where <type> is according to DBus specification:
 
+* <a>   : Array of ...
 * <s>   : String
-* <ay>  : Byte Array
-* <aay> : Array of Byte Arrays
-* <as>  : String Array
-* <ax>  : Array of signed 64 bit integer
-* <b>   : Boolean (0|1)
-* <x>   : Signed 64 bit integer
+* <y>   : Byte
+* <b>   : Boolean (false|true)
+* <x>   : Signed 64-bit integer (long)
+* <i>   : Signed 32-bit integer (int)
 * <>    : no return value
 
 Exceptions are the names of the Java Exceptions returned in the body field. They typically contain an additional message with details. All Exceptions begin with "org.asamk.Signal.Error." which is omitted here for better readability.
 
 Phone numbers always have the format +<countrycode><regional number>
 
-== Methods
+=== Control methods
```

Comment:
Thanks for documenting the Control interface!

Reply:
you're welcome